src/main/kotlin/no/nav/familie/tilbake/service/FaktaFeilutbetalingTjeneste.kt

```diff
@@ -0,0 +1,58 @@
+package no.nav.familie.tilbake.service
+
+import no.nav.familie.tilbake.behandling.domain.Behandling
+import no.nav.familie.tilbake.common.Periode
+import no.nav.familie.tilbake.faktaomfeilutbetaling.FaktaFeilutbetalingRepository
+import no.nav.familie.tilbake.faktaomfeilutbetaling.domain.FaktaFeilutbetaling
+import no.nav.familie.tilbake.service.modell.BehandlingFeilutbetalingFakta
+import no.nav.familie.tilbake.service.modell.HendelseTypeMedUndertypeDto
+import no.nav.familie.tilbake.service.modell.LogiskPeriode
+import no.nav.familie.tilbake.service.modell.LogiskPeriodeMedFaktaDto
+import org.springframework.stereotype.Service
+import java.time.LocalDate
+
+@Service
+class FaktaFeilutbetalingTjeneste(private val faktaFeilutbetalingRepository: FaktaFeilutbetalingRepository,
```

Comment:
Er ikkje dette noko Saikat måtte implementere for hentFakta-PR-en? Har han renama den?

Reply:
Det kan godt hende, men som jeg nevnte på stand up, så var dette kode jeg konverterte før Saikat begynte på sin branch. Merging av min og Saikat sin implementasjon bør være en egen PR for ikkeå fortsette å gjøre dobbeltarbeid.